### PR TITLE
Drop ARMv7 from docker

### PR DESCRIPTION
**Pull request by erikjohnston**
It's proving incredibly hard to build in CircleCI infra.

c.f. #9403 (and #9423)

### CODE REVIEW

File: changelog.d/9433.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Drop support for ARMv7 in Docker images.
```
Comment (clokep) [inline changelog.d/9433.misc:1]: Should this be a `.docker` or `.removal`?

Comment (erikjohnston) [reply changelog.d/9433.misc:1]: I keep forgetting those things exist

Comment (erikjohnston) [reply changelog.d/9433.misc]: I'm putting it in docker, fairly arbitrarily, but removal feels like its more for removing synapse features :shrug:

**Review by clokep — APPROVED** [review]
I think this is OK. I'm guessing we still want to do the multi-layered build here so that amd64 gets uploaded first? (I don't remember if both arm64 and arm/v7 were super slow.)

**Comment by erikjohnston** [comment]
> I think this is OK. I'm guessing we still want to do the multi-layered build here so that amd64 gets uploaded first? (I don't remember if both arm64 and arm/v7 were super slow.)

I think its probably safest to leave as is for now either way, tbh.

**Pull request merged**